### DISCUSSION TITLE
How we can make specific module functional in React Native?

### DISCUSSION THREAD

**Discussion by AwaisIbrar8** [react-native-community/discussions-and-proposals#645] [Ideas]
Let's say we have multiple modules in our App and we want to use some specific modules.

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. Yours seems more a question for [StackOverflow instead](https://stackoverflow.com/questions/tagged/react-native).

**Comment by yaghobi4398010** [comment]
my. ***@***.***

در تاریخ چهارشنبه ۳ مهٔ ۲۰۲۳،‏ ۱۴:۰۱ Lorenzo Sciandra <
***@***.***> نوشت:
